core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -72,12 +72,19 @@ oppia.directive('questionsList', [
             StateEditorService, QuestionUndoRedoService, UndoRedoService) {
           var ctrl = this;
           ctrl.currentPage = 0;
+          ctrl.skillIdsList = [];
 
           var _initTab = function() {
+            if (ctrl.getSkill()) {
+              ctrl.skillIdsList = [ctrl.getSkill().getId()];
```

Comment:
As an FYI you don't really need the "List" at the end (I don't think we generally do that).

Reply:
Done.